hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/s3guard/S3Guard.java

```diff
@@ -35,6 +35,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Preconditions;
+import org.apache.hadoop.fs.RemoteIterator;
```

Comment:
nit: leave the empty line

Reply:
there was no empty line here

Reply:
(add an empty line between the `com.google.` and `org.apahce.hadoop` imports. we group our imports for easier backporting)

Reply:
thanks for clarification. will do